src/methods.jl

```diff
@@ -134,10 +153,12 @@ flat_correct(frame::AbstractArray, flat_frame::AbstractArray; kwargs...) = flat_
 """
     trim(frame::AbstractArray, idxs)
 
-Trims the frame to remove the region specified by idxs.
+Trims the `frame` to remove the region specified by idxs.
 
 This function trims the array in a manner such that final array should be rectangular.
 The indices follow standard Julia convention, so `(:, 45:60)` trims all columns from 45 to 60 and `(1:20, :)` trims all the rows from 1 to 20.
+The function also supports FITS convention when indices are passed as strings, so `"[45:60, :]"` works equivalently as `(:, 45:60)` and `"[:, 1:20]"`
+works equivalently as `(1:20, :)`.
```

Comment:
I would keep this briefer- "we support FITS-style indices". That way you can simply address this the same way for the docstrings and in the documentation we can write a section for `Indexing Styles`.

This way you don't have to explain what FITS-style indices are in _every_ docstring that uses them.